backend/src/routes/helpers/passwordRegex.ts

```diff
@@ -0,0 +1,2 @@
+export const passwordRegex =
+	/^[a-z0-9~`!@#$%^&*()_\-+={[}\]|:;\\"'<,>.?/]{8,72}$/i;
```

Comment:
would be nice to have space in the list of allowed characters

Reply:
somehow it felt odd to have space. no reason not to so added.